Add steps to handle Azure MySQL foreign keys (#330)
seqeralabs/docs: changelog/seqera-enterprise/v23.3.mdx

**Custom deployment**:

- Apply the database migration:

  - If you use Azure Database for MySQL:
    1. Connect to the database as a role with sufficient privileges to drop a column and run the following query:

        ```sql
        ALTER TABLE tw_label_resource DROP COLUMN IF EXISTS my_row_id;
        ```

>     1. Run the `/migrate-db.sh` script provided in the `backend` container. This will migrate the database schema.

Reviewer: ```suggestion
    1. Run the `/migrate-db.sh` script provided in the `backend` container to migrate the database schema.
```

>   - If you use any other database, run the `/migrate-db.sh` script provided in the `backend` container. This will migrate the database schema.

Reviewer: ```suggestion
  - If you use any other database, run the `/migrate-db.sh` script provided in the `backend` container to migrate the database schema.
```
